cloud/api.rst

```diff
@@ -0,0 +1,167 @@
+
+.. code-block:: text
+
+   Authorization: Token API_TOKEN
+
+For security, all requests must be issued over https. Additionally, API requests are limited to five per minute.
+
+Create Formation
+================
+
+* Endpoint: POST https://console.citusdata.com/api/v1/formations/
+* Sample request payload:
+
+  .. code-block:: json
+
+    {
+       "name": "my-formation",
+       "region": "us-east-1",
+       "tune_attributes": {
+          "coordinator_tune": "t2.small",
+          "worker_tune": "t2.small",
+          "coordinator_ha": false,
+          "worker_ha": false,
+          "worker_count": "2",
+          "coordinator_disk_size": "100",
+          "worker_disk_size": "100"
+       }
+    }
+
+  Parameters:
+
+  * (mandatory) **name**: a non-empty string of your choice
+  * (mandatory) **region**: any of our :ref:`cloud_regions`
+  * (mandatory) **coordinator_tune**, **worker_tune**: a tune is any of these AWS machine types: t2.small, t2.medium, t2.large,  r3.large, r3.xlarge, r3.2xlarge, r3.4xlarge, r3.8xlarge r4.large, r4.xlarge, r4.2xlarge, r4.4xlarge, r4.8xlarge, r4.16xlarge, c4.xlarge, c4.2xlarge, c4.4xlarge, c4.8xlarge, c5.xlarge, c5.2xlarge, c5.4xlarge, c5.9xlarge. *(If you choose a tune from the t2.\* series, both the coordinator_tune and worker_tune must be same.)*
+  * (mandatory) **coordinator_disk_size**, **worker_disk_size**: size in gigabytes. May be any of the values 100, 512, 1024, 1536, 2048, or 2560.  *(If you choose a tune from the t2.\* series, the disk size must be 100.)*
+  * (mandatory) **worker_count**: numeric value less than or equal to 40
+  * **coordinator_ha**, **worker_ha**: whether to enable High Availability. Default is false.
+
+* Example:
+
+   .. code-block:: bash
+
+     curl -H "Authorization: Token $API_TOKEN" \
+          -H "Content-Type: application/json" \
+          --request POST \
+          -d '{"tune_attributes": { "coordinator_tune": "t2.small", "worker_tune": "t2.small", "worker_count": "2", "coordinator_disk_size": "100", "worker_disk_size": "100"}, "name": "my-formation", "region": "us-east-1"}' \
+          https://console.citusdata.com/api/v1/formations/
+
+* Response:
+
+  .. code-block:: json
+
+    {
+      "id": "0123456789abcdef0123456789a"
+    }
+
+List Formations
+===============
+
+* Endpoint: GET https://console.citusdata.com/api/v1/formations/
+* Example:
+
+  .. code-block:: bash
+
+    curl -H "Authorization: Token $API_TOKEN" \
+      https://console.citusdata.com/api/v1/formations/
+
+* Response:
+
+  .. code-block:: json
+
+    {
+      "formations": [
+        {
+          "id": "0123456789abcdef0123456789a",
+          "name": "my-formation",
+          "region": "us-east-1",
+          "created_at": "2019-01-01T00:00:00Z"
+        }
+      ]
+    }
+
+Formation Details
+=================
+
+* Endpoint: GET https://console.citusdata.com/api/v1/formations/:formation_id
```

Comment:
Maybe use bold on :formation_id at the end of the endpoint to make it more clear this endpoint uses id of the formation.